beetsplug/bandcamp/http.py

```diff
@@ -0,0 +1,34 @@
+import http.client
+from functools import lru_cache
+from html import unescape
+from urllib.parse import urlsplit
+
+from beets import __version__
+
+
+USER_AGENT = f"beets/{__version__} +https://beets.io/"
+
+
+class HTTPBadStatus(http.client.HTTPException):
+    """HTTP request ended with non OK status."""
+
+    def __init__(self, url, response):
+        super().__init__(f"HTTP GET to {url} failed: {response.status} {response.reason}")
+        self.url = url
+        self.response = response
```

Comment:
Do these two attributes need to be set?
```suggestion
```

Reply:
As far as I understand this exception won't be necessary if we utilize an off-the-shelf tool like `httpx`?